testapi.pm

```diff
@@ -35,6 +35,7 @@ use MIME::Base64 'decode_base64';
 use Scalar::Util 'looks_like_number';
 
 require bmwqemu;
+use constant OPENQA_LIBPATH => '/usr/share/openqa/lib';
```

Comment:
I wonder if there was a better way, but in the long run, since it's being cached if the lib is not found, shouldn't then be a major problem

Reply:
This is how it looks like currently when it can't find the module : http://e122.suse.de/tests/5576#step/slenkins_control/16

Yeah, it's not an elegant solution, wondering if at this point we could just add openqa libpath by default in @INC, but that means the whole os-autoinst would depend on openQA-common